website/notifications/__init__.py

```diff
@@ -0,0 +1 @@
+__author__ = 'Saman'
```

Comment:
We haven't really established a policy for designating authorship/ownership of parts of the codebase. This is a tricky area, and it only gets more complex as a project matures. For now, let's omit any designations of authorship aside from the git history.